sbin/prepareWorkspace.sh

```diff
@@ -286,8 +286,33 @@ updateOpenj9Sources() {
   # Building OpenJDK with OpenJ9 must run get_source.sh to clone openj9 and openj9-omr repositories
   if [ "${BUILD_CONFIG[BUILD_VARIANT]}" == "${BUILD_VARIANT_OPENJ9}" ]; then
     cd "${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[WORKING_DIR]}/${BUILD_CONFIG[OPENJDK_SOURCE_DIR]}" || return
+
+    if [ "$JAVA_FEATURE_VERSION" -eq 17 ]
+    then
+      if [[ $BUILD_ARGS == *"--bundle-openjceplus"* ]]; then
```

Comment:
I think we need to add a section in `config_init.sh` under `parseConfigurationArguments()`. Then we can `if` here based on `BUILD_CONFIG` instead of BUILD_ARGS which may not be available anymore?